jobs/dtd-catalog-total-load-exporter/src/services/read-model-queries.service.ts

```diff
@@ -0,0 +1,116 @@
+import { Attribute, Attributes, ReadModelClient, TENANTS_COLLECTION_NAME } from '@interop-be-reports/commons'
+import { z } from 'zod'
+import { EServiceQueryOutput } from '../models/eservice-result.model.js'
+
+export async function getEServices(readModel: ReadModelClient): Promise<Array<EServiceQueryOutput>> {
+  return await readModel.eservices
+    .aggregate([
+      {
+        $match: { 'data.descriptors.state': { $in: ['Published', 'Suspended'] } },
+      },
+      {
+        $lookup: {
+          from: TENANTS_COLLECTION_NAME,
+          localField: 'data.producerId',
+          foreignField: 'data.id',
+          as: 'producer',
+        },
+      },
+      {
+        $addFields: {
+          activeDescriptor: {
+            $filter: {
+              input: '$data.descriptors',
+              as: 'descriptor',
+              cond: {
+                $or: [{ $eq: ['$$descriptor.state', 'Published'] }, { $eq: ['$$descriptor.state', 'Suspended'] }],
+              },
+            },
+          },
+        },
+      },
+      {
+        $addFields: {
+          activeDescriptor: { $arrayElemAt: ['$activeDescriptor', 0] },
```

Comment:
we may have 
- descriptor1 Suspended
- descriptor2 Published
in this case the Suspended descriptor would be taken, is it ok?